### PR TITLE
bump rustfmt and rust toolchain

### PR DESCRIPTION
**Pull request by brotzeit**
I have problems installing rustfmt 0.2.15.

### CODE REVIEW

File: rust_src/remacs-sys/lib.rs [hunk 1]
```diff
@@ -936,7 +934,6 @@ pub struct Lisp_Frame {
     /// List of buffers that were viewed, then buried in this frame.  The
     /// most recently buried buffer is first.  For last-buffer.
     pub buried_buffer_list: Lisp_Object,
-
```
Comment (brotzeit) [inline rust_src/remacs-sys/lib.rs]: That's strange formatting...

**Comment by shaleh** [comment]
Their documentation is woefully behind reality.

**Comment by shaleh** [comment]
`blank_lines_lower_bound` and `blank_lines_upper_bound` perhaps?

**Comment by brotzeit** [comment]
```
clang: error: unable to execute command: Segmentation fault: 11

clang: error: dsymutil command failed due to signal (use -v to see invocation)
```

This is new isn't it ?

**Comment by shaleh** [comment]
Happened when Travis updated the OSX images. It is plaguing our builds and users of recent OSX versions.

**Comment by brotzeit** [comment]
I have tried `blank_lines_lower_bound` and `blank_lines_upper_bound` but it enforces those boundaries everywhere.

**Comment by shaleh** [comment]
Toggling the build

**Comment by shaleh** [comment]
Toggling build

**Comment by brotzeit** [comment]
:clap:

**Pull request merged**